src/popup/router/components/ButtonSubheader.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <Button
+    class="button-subheader"
+    :class="fill"
+    :to="to"
+    @click="$emit('click')"
+  >
+    <div class="box">
+      <span class="icon">
+        <slot />
+      </span>
+
+      <div class="message">
+        <span class="heading">{{ header }}</span>
+        <span class="description">{{ subheader }}</span>
+      </div>
+    </div>
+  </Button>
+</template>
+
+<script>
+export default {
+  props: {
+    to: { type: [String, Object], default: '' },
+    header: { type: String, default: '' },
+    subheader: { type: String, default: '' },
+    fill: {
+      type: String,
+      validator: (value) => ['primary', 'alternative'].includes(value),
+      default: 'primary',
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@use '../../../styles/variables';
+@use '../../../styles/typography';
+@use '../../../styles/mixins';
+
+.button-subheader {
+  background: variables.$color-bg-6;
+  border-radius: 10px;
+  margin: 16px;
+  height: 80px;
+  width: 328px;
```

Comment:
Please avoid setting fixed widths for elements. The spacing between the edge of the app and the button should be done with the use of paddings set on the parent.